diff --git a/Makefile b/Makefile
index 2341cf9..9b457df 100644
--- a/Makefile
+++ b/Makefile
@@ -90,13 +90,13 @@ install:	$(ALL)
 	install -m 0755 pig2vcd           $(DESTDIR)$(bindir)
 	install -m 0755 pigpiod           $(DESTDIR)$(bindir)
 	install -m 0755 pigs              $(DESTDIR)$(bindir)
-	if which python2; then python2 setup.py install; fi
-	if which python3; then python3 setup.py install; fi
+	if which python2; then python2 setup.py install --home=$(DESTDIR); fi
+	if which python3; then python3 setup.py install --home=$(DESTDIR); fi
 	install -m 0755 -d                $(DESTDIR)$(mandir)/man1
 	install -m 0644 *.1               $(DESTDIR)$(mandir)/man1
 	install -m 0755 -d                $(DESTDIR)$(mandir)/man3
 	install -m 0644 *.3               $(DESTDIR)$(mandir)/man3
-	ldconfig
+	#ldconfig
 
 uninstall:
 	rm -f $(DESTDIR)$(includedir)/pigpio.h
@@ -112,7 +112,7 @@ uninstall:
 	if which python3; then python3 setup.py install --record /tmp/pigpio >/dev/null; xargs rm -f < /tmp/pigpio >/dev/null; fi
 	rm -f $(DESTDIR)$(mandir)/man1/pig*.1
 	rm -f $(DESTDIR)$(mandir)/man3/pig*.3
-	ldconfig
+	#ldconfig
 
 $(LIB1):	$(OBJ1)
 	$(SHLIB) -o $(LIB1) $(OBJ1)
